charts/openebs/README.md

```diff
@@ -43,64 +43,63 @@ The following table lists the configurable parameters of the OpenEBS chart and t
 | `image.repository`                      | Specify which docker registry to use          | `quay.io/`                                |
```

Comment:
change the default value for image.repository to `""`, and also make this change in the values.yaml. 

As quay.io is going to be under maintenance on 13th and possibly have additional downtime in the coming weeks.

Reply:
Done